### ISSUE TITLE
[Bug report] - SOUND SYNC: COULD NOT FIND DATA ON BNC2

### ISSUE THREAD

**Issue by julius-benson** [int-brain-lab/iblenv#120]
I've stopped hearing trial-start beeps on two of our behavioral rigs. One of them has been giving us this warning for a week or two now, and another rig just started giving us this error just today after Windows updated over the weekend. I thought it might be an issue with the sound card and just planned to replace one of them, but now that a second rig has stopped giving beeps, I'm wondering if it's a problem with the way Windows has its sound settings configured.

**Comment by mayofaulkner** [comment]
Hi Julius, have you checked all of the xonar sound card settings to make sure that they are on the correct settings? Sometimes after windows update it changes these. You can find the instructions for configuring the settings on page 36 or this rig building document https://docs.google.com/document/d/1mEOYdrmwH4W0abhYO_OmtRCfwJCL8AQf5K99eVh6S0U/edit#heading=h.uil3zjwlyvy7

**Comment by GaelleChapuis** [comment]
From @nbonacchi :
- Windows update(s) may cause your sound card to stop being recognized by windows, and consequently the card driver cannot be updated/installed.
- The offending updates currently known as per Asus website are KB4579311 and KB4577671 (you might want to confirm the numbers on your windows machine). Asus has a link explaining the steps here: https://www.asus.com/support/FAQ/1044486
- Uninstall the windows update(s), reinstall latest card driver, and reinstall the windows update(s).

**Comment by julius-benson** [comment]
Uninstalling KB4579311 worked on one of the 2 broken rigs. I was able to install Xonar AE 1.1.18 on that one and select the correct output device (we use the headphones port) to get the trial-start beeps again.

On the other broken rig, trying to uninstall KB4579311 does not work. Running `wusa /uninstall /kb:4579311` gives us error code `0x800f0905`. I tried uninstalling the latest update from the reboot menu (holding shift while clicking restart) but the computer just hung on a black screen and I forced it off the next morning. On reboot, KB4579311 still refuses to uninstall. My guess is that a more recent update depends on it, but none of my other updates give me the option to uninstall them.
